File: angular/src/app/main/payroll/shared/dto/monthlyCpr-dto.ts

export class MonthlyCPRDto {
    id:number|undefined;
    salaryYear: number;
    salaryMonth: number;
    cprNumber: string;
    cprDate: Date | string | null;
    amount: number | null;
    remarks: string;
    active: boolean;
    audtUser: string;
    audtDate: Date | string | null;
    createdBy: string;
    createDate: Date | string | null;
    }

export class GetMonthlyCprForViewDto {
    monthlyCprDto: MonthlyCPRDto
}
export class GetMonthlyCprEditOutput {
    monthlyCPR: MonthlyCPRDto
}

export class PagedResultDtoMonthlyCpr {
    totalCount: number;
    items: GetMonthlyCprForViewDto[]
}
export class CreateOrEditMonthlyCprDto {
    id:number|undefined;
    salaryYear: number;
    salaryMonth: number;
    cprNumber: string;
    cprDate: Date | string | null;
    amount: number | null;
    remarks: string;
    active: boolean;
}

File: aspnet-core/src/ERP.Application.Shared/PayRoll/MonthlyCPR/Dtos/MonthlyCPRDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.PayRoll.MonthlyCPR.Dtos
{
    public class MonthlyCPRDto : EntityDto
    {
        public short SalaryYear { get; set; }

        public short SalaryMonth { get; set; }

        public string CPRNumber { get; set; }

        public DateTime? CPRDate { get; set; }

        public double? Amount { get; set; }

        public string Remarks { get; set; }

        public bool Active { get; set; }

        public string AudtUser { get; set; }

        public DateTime? AudtDate { get; set; }

        public string CreatedBy { get; set; }

        public DateTime? CreateDate { get; set; }

    }
}